traveler-ios-ui/TravelerKitUI/ViewControllers/AvailabilityViewController.swift

```diff
@@ -65,17 +85,54 @@ open class AvailabilityViewController: UIViewController {
         }
     }
 
+    @IBAction func didSelectPreviousMonth(_ sender: Any) {
+        guard let volatileDate = Calendar.current.date(byAdding: .month, value: -1, to: selectedDate) else {
+            Log("Unknown error getting the date of previous month.", data: nil, level: .error)
+            return
+        }
+
+        selectedDate = volatileDate
+        if let calendarVC = calendarViewControllers[selectedDate] {
+            pageController?.setViewControllers([calendarVC], direction: .reverse, animated: true, completion: nil)
+        } else {
+            let storyboard = UIStoryboard(name: "BookingItem", bundle: Bundle(for: AvailabilityViewController.self))
+            let calendarVC = storyboard.instantiateViewController(withIdentifier: "calendarVC") as! AvailabilityCalendarViewController
+            calendarVC.delegate = self
+            calendarVC.product = product
+            calendarVC.representingDate = selectedDate
+            calendarViewControllers[selectedDate] = calendarVC
+            pageController?.setViewControllers([calendarVC], direction: .reverse, animated: true, completion: nil)
+        }
+    }
+
+    @IBAction func didSelectNextMonth(_ sender: Any) {
+        guard let volatileDate = Calendar.current.date(byAdding: .month, value: 1, to: selectedDate) else {
+            Log("Unknown error getting the date of next month.", data: nil, level: .error)
+            return
+        }
+
+        selectedDate = volatileDate
+        if let calendarVC = calendarViewControllers[selectedDate] {
+            pageController?.setViewControllers([calendarVC], direction: .forward, animated: true, completion: nil)
+        } else {
+            let storyboard = UIStoryboard(name: "BookingItem", bundle: Bundle(for: AvailabilityViewController.self))
+            let calendarVC = storyboard.instantiateViewController(withIdentifier: "calendarVC") as! AvailabilityCalendarViewController
```

Comment:
Do NOT use a PagingViewContorller

That VC is meant for finite set of VCs, not for piggybacking off of its swipe gesture